src/Compilers/CSharp/Portable/Symbols/Source/ModifierUtils.cs

```diff
@@ -19,9 +19,10 @@ internal static class ModifierUtils
             DeclarationModifiers allowedModifiers,
             Location errorLocation,
             BindingDiagnosticBag diagnostics,
-            out bool modifierErrors)
+            out bool modifierErrors,
+            bool isOrdinaryMethod = false)
         {
-            var result = modifiers.ToDeclarationModifiers(diagnostics.DiagnosticBag ?? new DiagnosticBag());
+            var result = modifiers.ToDeclarationModifiers(diagnostics.DiagnosticBag ?? new DiagnosticBag(), isOrdinaryMethod: isOrdinaryMethod);
```

Comment:
I'd feel better about grouping this parameter with the other "contextual flags": `isForTypeDeclaration`, `isForInterfaceMember`, and making it required. Looks like we have only about 8 call sites to update. (BTW, is the argument for `isForTypeDeclaration` ever `true`?)

Reply:
`isForTypeDeclaration` appears to always be `false`. Let me clean that up while here.